README.md

```diff
@@ -41,6 +41,7 @@ If you want to use a package manager:
 - [Chocolatey](https://chocolatey.org/) users can use `choco install kubernetes-helm`.
 - [Scoop](https://scoop.sh/) users can use `scoop install helm`.
 - [GoFish](https://gofi.sh/) users can use `gofish install helm`.
+- [Snapcraft](https://snapcraft.io/) users can use `snap install helm --classic`
```

Comment:
This probably should this be: `snap install helm`

Reply:
This appears correct according to https://github.com/snapcrafters/helm/blob/master/README.md

I believe the flag is intentional to handle a case in snap’s configuration. Not sure of the reason behind it, but that flag is required to install helm last time I tried.

Reply:
@bacongobbler is correct, this snap requires the `--classic` flag for the "classic" confinement. You may find more details here - https://snapcraft.io/docs/snap-confinement.

PS. It also requires the superuser power to install, but I haven't added `sudo` following the consistency with the installation methods listed above.

Reply:
OK, my bad, thought it was a typo